ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -53,23 +54,31 @@ - (void)acquireToken:(NSString *)apiId
     [self ensureRequest];
     NSString* telemetryRequestId = [_requestParams telemetryRequestId];
     
-    __block NSString* log = [NSString stringWithFormat:@"##### BEGIN acquireToken%@ (authority = %@, resource = %@, clientId = %@, idtype = %@) #####",
-                             _silent ? @"Silent" : @"", _requestParams.authority, _requestParams.resource, _requestParams.clientId, [_requestParams.identifier typeAsString]];
-    AD_LOG_INFO_PII(_requestParams.correlationId, @"userId = %@", _requestParams.identifier.userId);
+    NSString *logMessage = [NSString stringWithFormat:@"%@ idtype = %@", _silent ? @"Silent" : @"", [_requestParams.identifier typeAsString]];
+    NSString *logMessagePII = [NSString stringWithFormat:@"resource = %@, clientId = %@, userId = %@", _requestParams.resource, _requestParams.clientId, _requestParams.identifier.userId];
+    if ([ADAuthorityUtils isKnownHost:_requestParams.authority]) {
+        logMessage = [NSString stringWithFormat:@"%@ authority: %@", logMessage, _requestParams.authority];
+    } else {
+        logMessagePII = [NSString stringWithFormat:@"%@ authority: %@", logMessagePII, _requestParams.authority];
+    }
+    
+    AD_LOG_INFO(_requestParams.correlationId, @"##### BEGIN acquireToken %@ #####", logMessage);
+    AD_LOG_INFO_PII(_requestParams.correlationId, @"##### BEGIN acquireToken %@ %@#####", logMessage, logMessagePII);
     
     ADAuthenticationCallback wrappedCallback = ^void(ADAuthenticationResult* result)
     {
         if (result.status == AD_SUCCEEDED)
         {
-            AD_LOG_INFO(result.correlationId, @"##### END %@ succeeded. #####", log);
+            AD_LOG_INFO(result.correlationId, @"##### END succeeded. %@ #####", logMessage);
+            AD_LOG_INFO(result.correlationId, @"##### END succeeded. %@ %@ #####", logMessage, logMessagePII);
         }
         else
         {
             ADAuthenticationError* error = result.error;
-            AD_LOG_INFO(result.correlationId, @"##### END %@ failed { domain: %@ code: %ld protocolCode: %@ ", log, error.domain, (long)error.code, error.protocolCode);
-            AD_LOG_INFO_PII(result.correlationId, @"errorDetails: %@ ", error.errorDetails);
+            AD_LOG_INFO(result.correlationId, @"##### END failed { domain: %@ code: %ld protocolCode: %@ %@ #####", error.domain, (long)error.code, error.protocolCode, logMessage);
+            AD_LOG_INFO_PII(result.correlationId, @"#### END failed { domain: %@ code: %ld protocolCode: %@ errorDetails: %@ %@ %@ #####", error.domain, (long)error.code, error.protocolCode, error.errorDetails, logMessage, logMessagePII);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 238 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)